app/src/main/java/org/sopt/and/designsystem/component/text/WhiteGrayText.kt

```diff
@@ -0,0 +1,31 @@
+package org.sopt.and.designsystem.component.text
+
+import androidx.compose.foundation.layout.Row
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.sp
+
+@Composable
+fun WhiteGrayText(
+    modifier: Modifier,
+    fontSize: Int,
+    whiteText: String,
+    grayText: String,
+) {
```

Comment:
modifer는 항상 기본값을 넣어주는것이 좋습니다.
```suggestion
@Composable
fun WhiteGrayText(
    fontSize: Int,
    whiteText: String,
    grayText: String,
    modifier: Modifier = Modifier,
) {
```